packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -169,7 +171,10 @@
       "props-default-doc:@aws-cdk/aws-lambda.Permission.sourceArn",
       "docs-public-apis:@aws-cdk/aws-lambda.ResourceBindOptions",
       "docs-public-apis:@aws-cdk/aws-lambda.VersionAttributes",
-      "props-physical-name:@aws-cdk/aws-lambda.EventInvokeConfigProps"
+      "props-physical-name:@aws-cdk/aws-lambda.EventInvokeConfigProps",
+      "props-struct-name:@aws-cdk/aws-lambda.IScalableFunctionAttribute",
+      "construct-ctor-props-type:@aws-cdk/aws-lambda.ScalableFunctionAttribute",
+      "construct-interface-extends-iconstruct:@aws-cdk/aws-lambda.IScalableFunctionAttribute"
```

Comment:
Why exclude these?

Reply:
I get these linting errors -- would love advice for how to silence them if I shouldn't skip them in package.json. Looks like I would need to add `ScalableFunctionAttributeProps`, which seems redundant.

```bs
error: [awslint:props-struct-name:@aws-cdk/aws-lambda.IScalableFunctionAttribute] all constructs must have a props struct 
error: [awslint:construct-ctor-props-type:@aws-cdk/aws-lambda.ScalableFunctionAttribute] construct "props" type should use the props struct @aws-cdk/aws-lambda.ScalableFunctionAttributeProps
```

Reply:
I looked into this and the error is indeed thrown since  the `ScalableFunctionAttribute` which extends `BaseScalableAttribute` accepts `BaseScalableAttributeProps` and does not include `ScalableFunctionAttributeProps`. 

In `ScalableTaskCount`  class in ecs which implements autoscaling for ecs task an empty interface was added to avoid this lint error:
```
/**
 * The properties of a scalable attribute representing task count.
 */
export interface ScalableTaskCountProps extends appscaling.BaseScalableAttributeProps {

}
```
https://github.com/aws/aws-cdk/blob/e6dca529098e54c91f0706019b9ee06522ddb025/packages/%40aws-cdk/aws-ecs/lib/base/scalable-task-count.ts#L9

Seems like the exclusion is better than adding an empty interface? @eladb

Reply:
No, an empty interface is the correct way to solve this as it will allow us to add fields later without breaking users. That's why the lint rule exists.